lib/fathead/c_module_example/parse.py

```diff
@@ -0,0 +1,123 @@
+			   "http://en.cppreference.com/w/cpp/header/thread", "http://en.cppreference.com/w/cpp/header/mutex", "http://en.cppreference.com/w/cpp/header/shared_mutex",
+			   "http://en.cppreference.com/w/cpp/header/future", "http://en.cppreference.com/w/cpp/header/condition_variable", "http://en.cppreference.com/w/cpp/header/filesystem",
+			   "http://en.cppreference.com/w/cpp/io/basic_streambuf", "http://en.cppreference.com/w/cpp/io/basic_filebuf", "http://en.cppreference.com/w/cpp/io/basic_stringbuf",
+			   "http://en.cppreference.com/w/cpp/io/ios_base", "http://en.cppreference.com/w/cpp/io/basic_ios", "http://en.cppreference.com/w/cpp/io/basic_istream",
+			   "http://en.cppreference.com/w/cpp/io/basic_ostream", "http://en.cppreference.com/w/cpp/io/basic_iostream", "http://en.cppreference.com/w/cpp/io/basic_ifstream",
+			   "http://en.cppreference.com/w/cpp/io/basic_ofstream", "http://en.cppreference.com/w/cpp/io/basic_fstream", "http://en.cppreference.com/w/cpp/io/basic_istringstream",
+			   "http://en.cppreference.com/w/cpp/io/basic_ostringstream", "http://en.cppreference.com/w/cpp/io/basic_stringstream", "http://en.cppreference.com/w/cpp/io/manip",
+			   "http://en.cppreference.com/w/cpp/io/c", "http://en.cppreference.com/w/cpp/string/basic_string", "http://en.cppreference.com/w/cpp/string/basic_string_view",
+			   "http://en.cppreference.com/w/cpp/string/byte", "http://en.cppreference.com/w/cpp/string/multibyte", "http://en.cppreference.com/w/cpp/string/wide",
+			   "http://en.cppreference.com/w/cpp/container/array", "http://en.cppreference.com/w/cpp/container/vector", "http://en.cppreference.com/w/cpp/container/deque",
+			   "http://en.cppreference.com/w/cpp/container/list", "http://en.cppreference.com/w/cpp/container/forward_list", "http://en.cppreference.com/w/cpp/container/set",
+			   "http://en.cppreference.com/w/cpp/container/multiset", "http://en.cppreference.com/w/cpp/container/map", "http://en.cppreference.com/w/cpp/container/multimap",
+			   "http://en.cppreference.com/w/cpp/container/unordered_set", "http://en.cppreference.com/w/cpp/container/unordered_multiset", "http://en.cppreference.com/w/cpp/container/unordered_map",
+			   "http://en.cppreference.com/w/cpp/container/unordered_multimap", "http://en.cppreference.com/w/cpp/container/stack", "http://en.cppreference.com/w/cpp/container/queue",
+			   "http://en.cppreference.com/w/cpp/container/priority_queue"]
+
+def print_line(title, abstract):
+    
+    # Duplicate check
+	if(MODULE_DICT.has_key(title)):
+		return
+
+	MODULE_DICT[title] = 1
+
+	abstract = abstract.replace('\n','\\n')
+	abstract = abstract.replace('\t', '  ')
+	abstract = abstract.replace(u'\u2019','&#8217;')
+	abstract = abstract.replace(u'\u201c','&#8220;')
+	abstract = abstract.replace(u'\u201d','&#8221;')
+	abstract = abstract.replace(u'\xb6','')
+	abstract = abstract.replace(u'\xa0','')
+	abstract = abstract.replace(u'\u2013','&#8211;')
+	abstract = abstract.replace(u'\u2018','&#8216;')
```

Comment:
Why are you replacing these characters particularly? Can't there be more non-ASCII characters?